lang/golang/golang-package.mk

```diff
@@ -217,8 +216,8 @@ define GoPackage/Build/Configure
 				base=$$$$(basename $$$$dir) ; \
 				if [ -d $$$$dest/$$$$base ]; then \
 					case $$$$dir in \
-					*$(GO_PKG_PATH)/src/$(GO_PKG)) \
-						echo "$(GO_PKG) is already installed. Please check for circular dependencies." ;; \
+					*$(GO_PKG_PATH)/src/$(strip $(GO_PKG))) \
+						echo "$(strip $(GO_PKG)) is already installed. Please check for circular dependencies." ;; \
```

Comment:
How about sanitizing these variables at the beginning of golang-package.mk once and for all.

Reply:
They can be changed after golang-package.mk is included.

Reply:
For example, in #10154 `GO_PKG_BUILD_PKG` is set twice inside `Build/Compile`.

It would be great if everyone only set `GO_PKG_*` variables before including golang-package.mk, but then it would also be great is everyone didn't include any preceding or trailing whitespace :joy: